### ISSUE TITLE
Display current builds as soon as they are fetched

### ISSUE THREAD

**Issue by ferjm** [servo/buildbotstatus#2]
We are currently waiting until all the builds are fetched from the server before displaying any information on the screen.

The tool fetches the [current builds](https://github.com/servo/buildbotstatus/blob/master/app/scripts/buildbot_client.js#L48) before the cached ones. We could add a new callback on `BuildbotClient.fetchBuilds` to let the UI know when the current builds are ready to be shown.

**Comment by patgronowski** [comment]
@ferjm I see you self-assigned this issue and I was wondering if I could try to solve this issue.

**Comment by ferjm** [comment]
Yes please, go for it! Ask questions if anything is unclear. Thanks!

**Comment by ferjm** [comment]
@patgronowski how are things going with this issue? Please, let me know if you need any help.

**Comment by jdm** [comment]
I don't think it's worth spending time on this any more. We're transitioning away from buildbot to taskcluster, which allows us to use tools like https://treeherder.allizom.org/#/jobs?repo=servo-auto instead.
